server/schedulers/balance_test.go

```diff
@@ -1211,7 +1211,7 @@ func (s *testScatterRangeLeaderSuite) TestConcurrencyUpdateConfig(c *C) {
 		for {
 			select {
 			case <-ch:
-				break
+				//break
```

Comment:
change to `return`?

Reply:
> change to `return`?

No break is needed in the default case.

Reply:
I think it should be `return` here

Reply:
why comment out it? we need to stop it here.

Reply:
> why comment out it? we need to stop it here.

ineffective break statement. Did you mean to break out of the outer loop? (SA4011)

Reply:
> why comment out it? we need to stop it here.



> change to `return`?

Change to `return` is a good choice.

Reply:
Please update it.

Reply:
> Please update it.

Done.